sdpa_op.py

```diff
@@ -0,0 +1,177 @@
+import torch
+from torch._subclasses import FakeTensorMode
+from torch.fx.experimental.symbolic_shapes import ShapeEnv, DimDynamic
+from torch._functorch.eager_transforms import _unwrap_all_tensors_from_functional, _wrap_all_tensors_to_functional, functionalize
+import torch.nn.functional as F
+import torch.fx as fx
+from torch._ops import HigherOrderOperator
+from torch._C import DispatchKey, DispatchKeySet, _ExcludeDispatchKeyGuard
+import torch.utils._pytree as pytree
+from torch.fx.experimental.proxy_tensor import (
+    disable_proxy_modes_tracing,
+    make_fx,
+    ProxyTorchDispatchMode,
+    track_tensor_tree,
+)
+from torch.utils._python_dispatch import (
+    _get_current_dispatch_mode,
+    _pop_mode_temporarily,
+)
+from torch.fx.experimental.proxy_tensor import make_fx
+from torch._inductor.compile_fx import compile_fx_inner
+
+sdpa = HigherOrderOperator("sdpa")
```

Comment:
Add docs somewhere about what the inputs/outputs to this function are and what it does?